### ISSUE TITLE
[framework]: custom font loading does not prevent CDN font calls

### ISSUE THREAD

**Issue by MapTo0** [SAP/ui5-webcomponents#7867]
### Bug Description

Based on https://sap.github.io/ui5-webcomponents/playground/?path=/docs/docs-customizing-fonts--docs adding

```html
<style type="text/css" data-ui5-font-face=""></style>
```

should prevent loading of fonts, but fonts are still loaded.
I noticed that when I call `delete Document.prototype.adoptedStyleSheets` loading is prevented.

To workaround the problem I used the following script:
```js
const styleSheet = new CSSStyleSheet();

const content  = ``;
styleSheet.replaceSync(content);

styleSheet._ui5StyleId = "data-ui5-font-face";
document.adoptedStyleSheets = [...document.adoptedStyleSheets, styleSheet];
```

### Affected Component

framework

### Expected Behaviour

no fonts are loaded

### Isolated Example

any app

### Priority

Medium

### UI5 Web Components Version

latest

### Browser

Chrome, Edge, Firefox, Safari

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by IlianaB** [comment]
Hello @SAP/ui5-webcomponents-topic-core , please have a look at this issue.

Regards,
Iliana

**Comment by rama1990br** [comment]
Hi @ilhan007,
Please downport this fix to 1.17 cause we have a dependency on this for the following patch:
https://jira.successfactors.com/browse/PTCH-42819 and

**Comment by ilhan007** [comment]
The fix has been released with 1.17.4
